docs/web/tutorial-common-lifecycle-hooks.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Lifecycle hooks for starting Workflows.
+description: VertiGIS Studio Web - Learn what lifecycle hooks are available and when to use them.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import UseCaseCard from "../../src/components/UseCaseCard";
+import UseCaseContainer from "../../src/components/UseCaseContainer";
+
+This article will tell you about common lifecycle hooks within Web and when you should use them for starting workflows.
+
+# Services > App > Events >
+
+## Application Initializing
+
+Event gets executed when Web begins to initialize. Commands or workflows must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading. This can be useful for launching workflows that are going to impact the initial state of the application (such as initial layout).
```

Comment:
```suggestion
This event gets executed when Web begins to initialize. Commands or workflows used here must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading. This can be useful for launching workflows that are going to impact the initial state of the application (such as initial layout).
```

Reply:
The first sentence is still just a fragment... but resolving this comment anyway.